### PR TITLE
Remove win-ca/pem before publish & catch inject error

### LINKED ISSUE #3357: Installing for all users causes error: JavaScript error occurred in the main process: Uncaught exception: Error: EPERM: operation not permitted
**Describe the bug**

When I open Lens, I get this error: 

JavaScript error occurred in the main process

Uncaught exception: 
Error: EPERM: operation not permitted, open 'C:\Program Files\Lens\resources\app.asar.unpacked\node_modules\win-ca\pem\root.pem'

Error Lens

**To Reproduce**
1. Update to Lens 5.0.2, install for all users
2. Open Lens 
3. See error

**Expected behavior**
1. Update to Lens 5.0.2, install for all users
2. Open Lens
3. Do not see error

**Screenshots**
See attached 

**Environment (please complete the following information):**
- Lens Version: 5.0.2-latest
- OS: Windows 10 Version 21H1, OS build 19043.1083
- Installation method: Windows install

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

fixes https://github.com/lensapp/lens/issues/3357

### CODE REVIEW

**Comment by jim-docker** [comment]
I still see this:

Screen Shot 2021-07-12 at 7 18 46 PM

I ran:
```
rm -rf node_modules/win-ca/pem
make build
'./dist/OpenLens Setup 5.0.3-beta.1.1626124453802.exe'
```

Oh, maybe `make build reinstalls the pem`

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -47,13 +47,18 @@ jobs:
         env:
           GH_TOKEN: $(LENS_IDE_GH_TOKEN)
         displayName: Customize config
-      
+
       - script: make node_modules
         displayName: Install dependencies
-      
+
+      - base: |
```
Comment (jakolehm) [inline .azure-pipelines.yml]: Probably would make sense to do this within `make build`:

```Makefile
build: node_modules binaries/client
	yarn run npm:fix-build-version
	$(MAKE) build-extensions -B
	yarn run compile
ifeq "$(DETECTED_OS)" "Windows"
	rm -rf node_modules/win-ca/pem
	yarn run electron-builder --publish onTag --x64 --ia32
else
	yarn run electron-builder --publish onTag
endif
```

Comment (Nokel81) [reply .azure-pipelines.yml]: Sounds good will change

**Comment by Nokel81** [comment]
@jim-docker PTALA?

File: src/common/system-ca.ts [hunk 2]
```diff
@@ -33,5 +33,9 @@ if (isMac) {
 }
 
 if (isWindows) {
-  winca.inject("+"); // see: https://github.com/ukoloff/win-ca#caveats
+  try {
+    winca.inject("+"); // see: https://github.com/ukoloff/win-ca#caveats
```
Comment (jim-docker) [inline src/common/system-ca.ts:37]: this is documented as experimental

File: .azure-pipelines.yml [hunk 3]
```diff
@@ -47,13 +47,13 @@ jobs:
         env:
           GH_TOKEN: $(LENS_IDE_GH_TOKEN)
         displayName: Customize config
-      
+
       - script: make node_modules
-        displayName: Install dependencies
-      
+        displayName: Install dependencies=
```
Comment (jim-docker) [inline .azure-pipelines.yml]: what does the `=` do?

Comment (Nokel81) [reply .azure-pipelines.yml]: oops that was accidental when reverting the previous change.

File: src/common/system-ca.ts (hunk 2 above)
Comment (Nokel81) [reply src/common/system-ca.ts:37]: Do you think we should just not do it instead?

Comment (jim-docker) [reply src/common/system-ca.ts:37]: If this fix works without using `"+"` then yes. Otherwise it's risky depending on an experimental feature

Comment (Nokel81) [reply src/common/system-ca.ts:37]: I believe that it is a separate concern to this EPERM issue. I added this `try/catch` as just some extra safety.

**Review by jim-docker — APPROVED** [review]
No more error notification! Still took a few tries to connect to the cluster (oops warning) but I assume that's another issue.

**Comment by Nokel81** [comment]
yes that is another issue. We need to solve it but in a different PR.

**Pull request merged**